bundle/src/test/java/com/adobe/acs/commons/version/impl/EvolutionContextImplTest.java

```diff
@@ -0,0 +1,187 @@
+package com.adobe.acs.commons.version.impl;
```

Comment:
missing license header

Reply:
Can we add a check for this in code climate?

Reply:
Maybe. Can you create a separate issue for that and assign it to me. I'll take a look.